bindings/python/src/OpenSpaceToolkitAstrodynamicsPy/Trajectory/Orbit/Model/Kepler/COE.cpp

```diff
@@ -387,6 +387,37 @@ inline void OpenSpaceToolkitAstrodynamicsPy_Trajectory_Orbit_Model_Kepler_COE(py
             arg("anomaly_type")
         )
 
+        .def_static(
```

Comment:
might be nice to also provide an overloaded constructor here (and the cpp) which accepts a Celestial from which you can get the gravitationalModel and access these parameters.